packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -114,6 +109,20 @@ export interface ISummarizeOptions {
     readonly refreshLatestAck?: boolean;
 }
 
+/**
+ * Data required to update internal tracking state after receiving a Summary Ack.
+ */
+ export interface IRefreshSummaryAckOptions {
+    /** Handle from the ack's summary op. */
+    readonly proposalHandle: string | undefined;
+    /** Handle from the summary ack just received */
+    readonly ackHandle: string;
+    /** Reference sequence number from the ack's summary op */
+    readonly summaryRefSeq: number;
```

Comment:
Nit: `summaryRefSeq` => `summaryRefSeqNum` it may be a little easier to read. I didn't immediately think of reference sequence "number", granted I had enough context to figure it out.

Reply:
I considered doing it but perhaps I will do later as the summaryRefSeq is being used on other public methods and I would need to change those as well.